Hi.js

```diff
@@ -0,0 +1,2 @@
+let name = 'Matias'
+const question `hello` ${name}, `how is your day?'
```

Comment:
You need an `=` sign between `question and `hello`. Also, the backtick should for the entire template literal. This is how you would correct this:

```
const question = `hello ${name}, how is your day?`;
```

Notice the `;` as well. Though not required, I would encourage you to always end your statements with `;` because it indicates to the reader an end.

Reply:
Thank you